apps/nextjs/src/app/[locale]/manage/tools/docker/page.tsx

```diff
@@ -0,0 +1,21 @@
+import { Box } from "@mantine/core";
+
+import { api } from "@homarr/api/server";
+
+export default async function DockerPage() {
+  const { containers, timestamp } = await api.docker.getContainers();
+  const timeAgo = new Intl.RelativeTimeFormat("en", {
```

Comment:
how can I do that ? Actually I was thiking of making a function for the relative time function as it might be used elsewhere, it would probably need uthe user's prefered language. Intl is a browser-specific API.

-> Do you agree with the utils function/hook if so, where should I put it?
Maybe a hook like `useRelativeTime();`

Reply:
Take a look at https://day.js.org/docs/en/plugin/relative-time
We already have included the dayjs translation by importing it in the `en.ts` file

Reply:
Is there a way to quickly get the users preferences so that I can load the correct language ?

Reply:
It's accessable through the params of next js (on client with `useParams<{locale: string}>`)